### PR TITLE
48358_add_progress_bars_in_trainee_home

### PR DESCRIPTION
**Pull request by zenith19**

### CODE REVIEW

File: app/helpers/courses_helper.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+module CoursesHelper
+
+  def find_progress sub_id
+    @courses_subject = CoursesSubject.find_by_course_id_and_subject_id(@course.id,sub_id)
```
Comment (abdullahallmehedi) [inline app/helpers/courses_helper.rb]: ()

File: app/helpers/courses_helper.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+  def find_progress sub_id
+    @courses_subject = CoursesSubject.find_by_course_id_and_subject_id(@course.id,sub_id) 
+    @tasks = @courses_subject.tasks
+    total_tasks_no = @tasks.length
+    completed_task_no = 0
+    percent = 0
+    completed = []
+    @tasks.each do |task|
+      completed = UsersTask.find_by_user_id_and_task_id(current_user.id, task.id)
```
Comment (abdullahallmehedi) [inline app/helpers/courses_helper.rb]: ()

File: app/views/courses/show.html.erb [hunk 3]
```diff
@@ -1,33 +1,46 @@
...
+    <th><%= t ".subj_name" %></th>
+    <th><%= t ".subj_desc" %></th>
+    <th><%= t ".progress_bars" %></th>
+  </thead>
+  <tbody>
+    <% @subjects.each do |subject| %>
+      <tr>
+        <td><%= link_to "#{subject.name.camelize}", 
+          subject_path(subject.id) %></td>
+        <td><%= subject.description.truncate(50) %></td>
```
Comment (abdullahallmehedi) [inline app/views/courses/show.html.erb]: ()

**Comment by abdullahallmehedi** [comment]
ok.

File: app/views/courses/show.html.erb [hunk 4]
```diff
@@ -1,33 +1,46 @@
...
+<table class="table table-striped">
+  <thead>
+    <th><%= t ".subj_name" %></th>
+    <th><%= t ".subj_desc" %></th>
+    <th><%= t ".progress_bars" %></th>
+  </thead>
+  <tbody>
+    <% @subjects.each do |subject| %>
+      <tr>
+        <td><%= link_to "#{subject.name.camelize}",
```
Comment (chuanhtuan) [inline app/views/courses/show.html.erb]: <%= link_to "#{subject.name.camelize}", subject %>

File: app/helpers/courses_helper.rb [hunk 5]
```diff
@@ -0,0 +1,42 @@
+module CoursesHelper
+
+  def find_progress sub_id
```
Comment (chuanhtuan) [inline app/helpers/courses_helper.rb]: no need write this method
the progress of course is the status of subjects of course
I think, It's very simple :(

Comment (zenith19) [inline app/helpers/courses_helper.rb]: Ok, I won't use this method for calculate the course percentage but I need this method for calculating the percentage of individual subject based on tasks.

**Pull request merged**